R/learner_prioritylasso_regr_priority_lasso.R

```diff
@@ -6,6 +6,9 @@
 #' Patient outcome prediction based on multi-omics data taking practitioners’ preferences into account.
 #' Calls [prioritylasso::prioritylasso()] from \CRANpkg{prioritylasso}.
 #'
+#' @section Custom mlr3 parameters:
```

Comment:
are these initial values or custom paramters? (custom parameters are those not present in the upstream learner like mtry.ratio for ranger)